### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by nschulux**
Translations of IMPEX , German and SWISS KBD , Printer PSCRIPT AND DUMB.

VIDEO BUT MUST BE UPDATED GEOS .INI  PLEASE TEST BEFORE ADD.

### CODE REVIEW

**Comment by jirkakunze** [comment]
What needs to be updated in geos.ini and why?

**Comment by nschulux** [comment]
IF we translate Videodriver this should be updated in Geos .ini 
This is also for Mouse and keyboard .
If not Geos can't find Graphics adapter !

From my Test Geos.ini with translated Drivers
[screen 0]
device = VESA kompatible Super VGA: 800x600 16 Farben
driver = Super VGA 800x600 16-clr
[mouse]
device = Windows-Maus-UnterstŸtzung ist installiert.
driver = GENMOUSE.GEO
info = 16384
[keyboard]
driver = Swiss-German Extended Keyboard
device = Schweizer-Deutsche erweiterte Tastatur
[printer]
numPrinters = 1
count = 1
defaultPrinter = 0
printers = GhostScript Software RIP Farbe (PostScript) in Datei
[GhostScript Software RIP Farbe (PostScript) in Datei]
type = 0
driver = PostScript driver
device = GhostScript Software RIP Farbe (PostScript)
port = FILE
intermittent = false

**Comment by bluewaysw** [comment]
ResEdit batch translation exepexts all translations file to live within one and the same folder. Sub folders are not allows (as far as I know). Please remove the folders and put all translation files into the german folder. Thx @nschulux

**Comment by nschulux** [comment]
All done.
But no Resedit Translation this results in Error KR-09 if I want to create a new Translationfile. This lies to the translation. But I can't find out why!
I have retried but no change !

**Review by bluewaysw — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (nschulux) [reply .vscode/settings.json:1]: I can't delete file , how to do ?

[the remark this answers is not in the document]
Comment (nschulux) [reply .vscode/settings.json:1]: vscode deleted !

**Comment by nschulux** [comment]
I have deleted .vscode/settings.json !

**Review by bluewaysw — APPROVED** [review]

**Comment by bluewaysw** [comment]
Ini file updates could be coded as following:

```
[screen 0]
PCDEMO(device = VGA: 640x480 16-color)
PCDEMO(driver = EC-dos(vga.geo))
GERMAN(NTDEMO(device = VESA kompatible SuperVGA: 800x600 64K Farben))
AMENGLISH(NTDEMO(device = VESA Compatible SuperVGA: 800x600 64K-color))
NTDEMO(driver = EC-dos(vga16.geo))
```

in the geos.ini template file.

**Pull request merged**